src/radio.cpp

```diff
@@ -2,16 +2,16 @@
 
 #define RADIO_CHANNEL_YAW_RATE 0
 #define RADIO_CHANNEL_PITCH 1
-#define RADIO_CHANNEL_THROTTLE 2
-#define RADIO_CHANNEL_ROLL 3
-#define RADIO_CHANNEL_VRA 5
-#define RADIO_CHANNEL_VRB 7
+#define RADIO_CHANNEL_ROLL 2
+#define RADIO_CHANNEL_THROTTLE 3
+#define RADIO_CHANNEL_VRA 4
+#define RADIO_CHANNEL_VRB 5
 #define RADIO_CHANNEL_VRC 6
-#define RADIO_CHANNEL_VRE 11
-#define RADIO_CHANNEL_SWITCH_A 9
-#define RADIO_CHANNEL_SWITCH_B 8
-#define RADIO_CHANNEL_SWITCH_C 4
-#define RADIO_CHANNEL_SWITCH_D 10
+#define RADIO_CHANNEL_VRE 7
+#define RADIO_SWITCH_A_BIT 0b01000
```

Comment:
Add documentation to explain what these binary numbers are

Reply:
Done.